src/GameState/Board.tsx

```diff
@@ -1,48 +1,56 @@
-import { Stack, HStack, Box } from 'native-base';
+import { Box, View } from 'native-base';
 import React from 'react';
-import { BOARD_GREEN, BOARD_GREY } from './boardColors';
+import { BOARD_GREEN } from './boardColors';
+import { convertBoardToBoardCoordinate } from './convertBoardToBoardCoordinate';
 import { Marble } from './Marble';
 
 export const Board = (props: any) => {
   const board = props.board;
-  const boardSize = props.preview ? 4 : 12;
+  const boardSize = props.preview ? 32 : '360px';
```

Comment:
What's the difference between 32 and 360px?

Reply:
Basically we used the sizes provided by Native Base. 
But it wasn't enough for us. So I unified in px